.github/workflows/ci.yaml

```diff
@@ -81,6 +81,10 @@ jobs:
             twisted-version: "latest"
             experimental: false
 
+          - python-version: "3.13"
```

Comment:
I've rebased my PR and it is still failing.
Shouldn't I specify python 3.13.0-rc.2 here or just wait until 3.13 is available (around beginning of October)?

Reply:
It looks like `3.13-dev` is the way to go:

https://github.com/twisted/twisted/blob/6bd87d34cc9e1fa562c92779e66fbdfed0e15737/.github/workflows/test.yaml#L140